### PR TITLE
Remove rhai for now.

### PR DESCRIPTION
**Pull request by o0Ignition0o**
The rhai plugin isn't quite in the shape we would like it to be.
We therefore remove it for this release, and we will iterate on it before we reenable it.

### CODE REVIEW

File: apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap [hunk 1]
```diff
@@ -143,6 +146,58 @@ expression: "&schema"
...
+          "minimum": 0.0,
+          "nullable": true
+        },
+        "tls_config": {
+          "$ref": "#/definitions/TlsConfig",
+          "nullable": true
+        }
+      },
+      "additionalProperties": false
+    },
```
Comment (abernix) [inline apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap:200]: What are all these changes to the snapshots about Grpc and Otel and stuff?

File: apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap [hunk 2]
```diff
@@ -298,6 +524,113 @@ expression: "&schema"
...
+      "properties": {
+        "collector": {
+          "default": "https://127.0.0.1:50051",
+          "type": "string"
+        },
+        "external": {
+          "type": "boolean"
+        },
+        "listener": {
+          "default": "127.0.0.1:50051",
```
Comment (abernix) [inline apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap:542]: And spaceport?

**Review by Geal — APPROVED** [review]

File: apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap (hunk 2 above)
Comment (o0Ignition0o) [reply apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap:542]: very good question, checking

File: apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap (hunk 1 above)
Comment (o0Ignition0o) [reply apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap]: a676ff45 got the root cause and fixed the test!

File: apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap (hunk 2 above)
Comment (o0Ignition0o) [reply apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap]: a676ff45 got the root cause and fixed the test!

**Review by BrynCooke — APPROVED** [review]

**Pull request merged**